src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
+* For Openshift, Lighthouse CoreDNS metrics are integrated with other CoreDNS metrics
+* KIND deployments support OVS CNI
+* `submariner_connections` is renamed to `submariner_requested_connections` in Prometheus for OpenShift clusters
+* Clusters can be connected with only Lighthouse and no dataplane connectivity via subctl
+* Added support for Flannel in k3s deployments
```

Comment:
?? where does this one come from, that's been there since we adopted submariner :)

Reply:
I think this is about the SUBMARINER_NETWORK_PLUGIN discovery fix from Masaki.

Reply:
Yes. This issue https://github.com/submariner-io/submariner/issues/1116